dataPipelines/gc_thumbnails/utils.py

```diff
@@ -0,0 +1,31 @@
+from pathlib import Path
+import fitz
+
+class ThumbnailsCreator:
+    def __init__(
+        self,
+        file_name,
+        output_directory
+    ):
+        self.file_name = file_name
+        self.output_directory = output_directory
+
+    def generate_png(self):
+        p = Path(self.output_directory)
+        if not p.exists():
```

Comment:
better to use `p.mkdir(exist_okay=True)`, since that's already available in `pathlib.Path`